Conceptpower+Spring/src/test/java/edu/asu/conceptpower/app/core/impl/RequestsManagerTest.java

```diff
@@ -19,22 +27,43 @@
  
     @InjectMocks
     private RequestsManager requestManager;
- 
+    
+    private ReviewRequest request = new ReviewRequest();
+    
     @Before
     public void setup() {
         MockitoAnnotations.initMocks(this);
         
+        List<ReviewRequest> responseList = new ArrayList<>();
+        ReviewRequest updatedRequest = new ReviewRequest();
+        request.setId("REVIEW1234-thisissupposedtobesecret");
```

Comment:
order of lines is very confusing. keep the things together that belong together: updated request with the rest of the updatedrequest lines, response list initialization below, where it's needed, request separate from that.

Reply:
also, id's are not secret, so no reason to call it that.

Reply:
?